.github/workflows/assets-test.yaml

```diff
@@ -39,11 +35,19 @@ jobs:
   check-execution-context:
     uses: Azure/azureml-assets/.github/workflows/check-execution-context.yaml@main
 
+  check-directory-file-changes:
+    needs: check-execution-context
+    if: fromJSON(needs.check-execution-context.outputs.continue)
+    uses: Azure/azureml-assets/.github/workflows/check-changed-files.yaml@update_more_gates  # TODO: Update this to @main later
```

Comment:
TODO: Delete test files and update to using check-directory-file-changes action from main branch

Reply:
updated